tensorflow/core/kernels/data/tensor_slice_dataset_op_test.cc

```diff
@@ -76,17 +76,31 @@ TestCase PlainTensorTestCase() {
           {DatasetOpsTestBase::CreateTensor<int64>(TensorShape({2}), {1, 2}),
```

Comment:
In addition to the test here, could you also add a Python-level test to `from_tensor_slices_test.py`? Thanks.

Reply:
done